ffs/api/api.go

```diff
@@ -224,7 +276,12 @@ func (i *API) Replace(c1 cid.Cid, c2 cid.Cid) (ffs.JobID, error) {
 		return ffs.EmptyJobID, fmt.Errorf("getting replaced cid config: %s", err)
 	}
 	cfg.Cid = c2
-	jid, err := i.sched.PushReplace(i.iid, i.cfg.WalletAddr, cfg, c1)
+
+	if err := i.ensureValidColdCfg(cfg.Cold); err != nil {
```

Comment:
For `Replace` and `PushConfig` we want to be sure the wallet address specified in the config is one actually managed by this instance (when necessary)